html/docs/include/functions/xdebug_stop_code_coverage

```diff
@@ -1,11 +1,11 @@
 = xdebug_stop_code_coverage
 Stops code coverage
 void
-[int cleanup=true]
+[int cleanup=1]
 FUNC_CODE_COVERAGE
 
 TXT:
 This function stops collecting information, the information in memory will
-be destroyed. If you pass "false" as argument, then the code coverage
+be destroyed. If you pass 0 as an argument (or a "false" value if not using strict types), then the code coverage
```

Comment:
I probably would only have changed `"false"` to `0` on this line. Would you mind doing that?

Reply:
Makes sense to me. My original intent was to keep a sense of backward compatibility for those not using strict typing, but that doesn't align with community best practices. 

Updated.